boards/vcc-gnd/yd_esp32/doc/index.rst

```diff
@@ -233,7 +233,7 @@ RGB LED
 =======
 
 The board contains an addressable RGB LED (`XL-5050RGBC-WS2812B`_), driven by GPIO16.
-Here is an example of how to test it using the :zephyr:code-sample:`led-ws2812` application.
+Here is an example of how to test it using the :zephyr:code-sample:`led-strip` application.
 
 .. zephyr-app-commands::
    :zephyr-app: samples/drivers/led_ws2812
```

Comment:
Please, replace the "samples/drivers/led_ws2812" above with "samples/drivers/led_strip".

And you need to fix the `boards/shields/adafruit_neopixel_grid_bff/doc/index.rst` file as well.

Reply:
Same as above.